tests/x11/seahorse.pm

```diff
@@ -32,9 +32,9 @@ sub run {
     send_key "ret";                                   # Next field (confirm PW)
     type_password;                                    # Re-type user password
     send_key "ret";                                   # Confirm password
-    assert_and_click "seahorse-default_keyring", 'right';    # right click the new keyring
-    assert_and_click "seahorse-set_as_default";              # Set the new keyring as default
-    send_key "alt-f4";                                       # Close seahorse
+    assert_and_click "seahorse-default_keyring", 'right', 60;    # right click the new keyring
```

Comment:
How sure we are that it helps? How many runs have you performed?

Reply:
over 20 times